src/pages/login/styles.scss

```diff
@@ -117,6 +117,15 @@
 }
 
 hr {
-  margin: 1rem;
+  margin: 0.5rem 1rem 1rem 1rem;
   width: 7.5rem;
 }
+
+.kep-login-facebook {
+  background-color: var(--color-white) !important;
+  border: none !important;
+  color: var(--color-black) !important;
+  font-weight: 900 !important;
+  font-size: 15px !important;
```

Comment:
Be careful with the `!important`, it can causes problems in the future if you overuse it. 
Do you really need it here?